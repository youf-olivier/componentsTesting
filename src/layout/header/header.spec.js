import React from "react";
import { render } from "@testing-library/react";
import Component from "./header";

describe("Header test suite", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
