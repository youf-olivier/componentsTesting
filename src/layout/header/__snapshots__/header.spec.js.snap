// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test suite renders correctly 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <div
      class="header__logo-container"
    >
      <img
        alt="logo axa"
        class="logo"
        src="https://www.axa.fr/content/dam/logo/logo-axa.svg"
      />
    </div>
    <span
      class="header__name"
    >
      Ma belle application
    </span>
  </header>
</DocumentFragment>
`;
